### ISSUE TITLE
Support for comment character

### ISSUE THREAD

**Issue by pylessard** [bialix/intelhex#25]
Altough it is not normally sppirted by the .hex format, many vendors embed comments in their .hex file, which make them impossible to parse by IntelHex unless pre-clean programmatically.

It would be nice to be able to specify an optional comment character (;, #, --) that would make the parser ignore any line starting with it.

Or mayve ignore everything not starting with a record delimiter.

**Comment by bialix** [comment]
Sorry for not working on your issue. I'm looking for a new maintainer for Python IntelHex project. I hope someone will help.

**Comment by tofkamp** [comment]
I agree with pylessard, add an option to ignore every lien not starting with ":", or skip any error.
Any logging captured will have some junk characters at the beginning and end.
